admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -188,17 +192,45 @@ Select a different default Encryption module::
 
  occ encryption:set-default-module [Module ID]. 
  
-The [module ID] is taken from the ``encryption:list-modules`` command. 
+The [module ID] is taken from the ``encryption:list-modules`` command.
+
+Encrypt all data files for all users. For performance reasons, when you enable 
+encryption on an ownCloud server only new and changed files are encrypted. This 
+command gives you the option to encrypt all files. You must first put your 
+ownCloud server into single-user mode to prevent any user activity until 
+encryption is completed::
+
+ occ encryption:encrypt-all
+
+Decrypt all user data files, or optionally a single user::
+ 
+ occ encryption:decrypt-all [username]
+
+Move keys to a different folder::
+
+ occ encryption:change-key-storage-root
+ 
+View current location of keys::
+
+ occ encryption:show-key-storage-root
+ 
+Create a new master key. Use this when you have a single-sign on 
+infrastructure.  Use this only on fresh installations with no existing data, or 
+on systems where encryption has not already been enabled. It is not possible to 
+disable it::
+
+ occ encryption:enable-master-key
  
 See :ref:`encryption_label`  for detailed instructions on using ``occ``.
 
 Files Not Encrypted
 -------------------
 
-Only the data in your files is encrypted, and not the filenames or folder
-structures. These files are never encrypted:
+Only the data in the files in ``data/user/files`` is encrypted, and not the 
+filenames or folder structures. These files are never encrypted:
 
-- Old files in the trash bin
+- Files in the trash bin
+- Versions
```

Comment:
files in the trashbin and versions are encrypted from the point on where you use encryption. We just don't encrypt existing files in the trashbin and versions with the encrypt-all occ tool